.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,50 @@
+name: e2e_mock_provider_tests
+on: 
+  workflow_dispatch:
+    inputs:
+      registry:
+        description: 'Registry for pulling images'
+        required: true
+        default: 'gcr.io/k8s-staging-csi-secrets-store'
+      driverImageName:
+        description: 'Secret Store CSI driver image name'
+        required: true
+        default: 'driver'
+      crdImageName:
+        description: 'Secret Store CSI driver CRD image name'
+        required: true
+        default: 'driver-crds'
+      imageVersion:
+        description: 'image version for Secret Store CSI driver and CRDs'
+        required: true
+        default: 'v0.3.0'
```

Comment:
will we need to update this when publishing releases?

does https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/main/docs/RELEASE.md need to be updated?

Reply:
Good idea. Updated the doc